app/middleware/renderer.js

```diff
@@ -1,8 +1,8 @@
 const log = require('../lib/logger');
 const mapLink = require('../lib/mapLink');
 
-function notFound(req, res, msg) {
-  log.warn(msg, 'Not found (404)');
+function notFound(req, res) {
+  log.warn({ req }, 'Not found (404)');
```

Comment:
`msg` wasn't consistently used so it was sometimes just a function. But the loosing of the message is of no consequence. The log will now have the full request object and the message `Not found (404)`